### PR TITLE
docs(test): add docs on testing with specific testsuite

### LINKED ISSUE #24: [Task] Add guide on how to validate bundle with specific test suite
Right now `make bundle` did a simple validation on the generated bundle, but we could also want to validate a specific test suite, for example, `operatorframework`.

```bash
operator-sdk-v1.22.2_linux_amd64 bundle validate ./bundle --select-optional suite=operatorframework
```

Related to #12

### PR DESCRIPTION
**Pull request by tthvo**
Fixes #24 
Related to #15 
Related to #12 

Added docs on how to test with specific test suite. I think the useful one now is just `operatorframework` suite.

### CODE REVIEW

**Comment by tthvo** [comment]
And just some rewording for titles and comments :D

**Comment by tthvo** [comment]
Also change numeration way to start from 1. I like the 0 more somehow but I guess generally we should start at 1 for documentations :D

File: docs/cryostat_operator.md [hunk 1]
```diff
@@ -126,37 +125,46 @@ _Note: crc is known to be very CPU and memory intensive_
...
 	![operator-installed-seen-on-console](img/openshift-cryostat-bundle.png)  
 
-4. To stop all crc, deployments, and pods cleanly
+5. To stop all crc, deployments, and pods cleanly
 	```bash
-	$ crc stop
-
-	# All resources will still be on disk the next time crc is started, unless crc delete is run
-	$ crc delete # optional
+	$ crc stop. # All resources will still be on disk the next time crc is started, unless crc delete is run
```
[the remark this answers is not in the document]
Comment (maxcao13) [reply docs/cryostat_operator.md:158]: @tthvo

**Review by maxcao13 — APPROVED** [review]
LGTM!

**Pull request merged**